### PR TITLE
Improve display of funding

### PR DESCRIPTION
**Pull request by cthoyt**
This PR moves the funding information from the right-hand bar to a panel. It updates the field to add some more structure information, including URLs I curated for all of the existing annotations. Luckily, this only affects four ontologies: UBERON, KISAO, ECO, and NOMEN. I was also able to remove some redundant text from each of these ontologies' free text sections referring to content that is structured and now displayed nicely.

I believe that the updates to the metadata in these files are within the purview of the Ops committee to merge without consulting the responsible people of these ontologies.

Here's how it looks now:

Screen Shot 2022-09-01 at 18 08 50

### CODE REVIEW

File: ontology/nomen.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ contact:
 description: NOMEN is a nomenclatural ontology for biological names (not concepts).  It encodes the goverened rules of nomenclature.
 domain: information
 funded_by:
-- NSF-ABI-1356381
-- Species File Group
```
Comment (cthoyt) [inline ontology/nomen.md:18]: note that `Species File Group` does not contain any substantial information, other than the name of the group that developed the ontology, and is not itself a funding source the way this field is expected to be used.

File: ontology/nomen.md [hunk 2]
```diff
@@ -40,5 +40,3 @@ usages:
   user: https://taxonworks.org
 activity_status: active
 ---
-
-NOMEN is a nomenclatural ontology for biological names (not concepts).  See the <a href="https://github.com/SpeciesFileGroup/nomen">NOMEN homepage</a> for more info.
```
Comment (cthoyt) [inline ontology/nomen.md:44]: The homepage link is redundant of the structured metadata, then the remaining sentence doubled the short description, so this is removed.

File: ontology/uberon.md [hunk 3]
```diff
@@ -178,5 +182,4 @@ wikidata_template: https://en.wikipedia.org/wiki/Template:Uberon
 activity_status: active
 ---
 
-Uberon is an integrated cross-species ontology covering anatomical structures in animals. See the <a href="http://uberon.org">Uberon website</a> for more info, or read the <a
```
Comment (cthoyt) [inline ontology/uberon.md:181]: note that the link to the website and the publication are now redundant of the structured information contained within the frontmatter

File: ontology/eco.md [hunk 4]
```diff
@@ -63,5 +64,3 @@ ECO was originally created around the year 2000 to support gene product annotati
 
 ***
 For **advice on requesting new terms**, please see **[the Evidence & Conclusion Ontology wiki](https://github.com/evidenceontology/evidenceontology/wiki/New-term-request-how-to)**.
-
-This work is made possible by **[award number 1458400](http://www.nsf.gov/awardsearch/showAward?AWD_ID=1458400)** from the **US National Science Foundation's Division of Biological Infrastructure**.
```
Comment (cthoyt) [inline ontology/eco.md:67]: note that this is now redundant of the structured information, which is displayed much more nicely.

**Review by matentzn — APPROVED** [review]
This affects ontology metadata governed by

@mjy
@jonrkarr
@mgiglio99

I judge these changes to be:

- a great improvement to the visibility and transparency of funding source
- involving changes that do not significantly impact the information content of any of the pages.
- The only slide exception to the "insignificant" changes may be the removal of the group as a funding source, @mjy. This PR suggests to use the field to describe funding sources a bit more informative. I acknowledge that grants are by far not the only sources of funding - many ontologies are developed by organisational "internal" funding. Your ontology is the only one it seems that lists a group as a funding source - can we somehow substantiate that? I would say that it goes without saying that for all ontologies, the groups developing the ontology will support the ontology with resources that go beyond the original grant. So I judge mentioning the group as redundant (the group is reflected by the github org). For now, I approve to remove this entry, but please feel free to challenge this decision a

**Comment by matentzn** [comment]
Very beautiful @cthoyt. Thank you.

**Comment by matentzn** [comment]
I will leave this open till 7th September Wednesday next week, then we merge it.

**Comment by mjy** [comment]
@all I'm fine with changes that reflect the removal of group.  I will note that as an endoweded biodiversity informatics group my funding, which maintains the servers (at least for a little while longer) that the HAO is built in is somewhat of a more precise set of funding.  I completely agree with your interpretation however, I also do many other things, so let's simplify things!

File: ontology/nomen.md (hunk 1 above)
Comment (balhoff) [reply ontology/nomen.md:18]: The Species File Group has an endowment and so may be a funding source. @mjy should review.

Comment (balhoff) [reply ontology/nomen.md:18]: Oops I see Matt already responded this morning—never mind!

**Comment by matentzn** [comment]
I am changing my merge data to Tuesday 6th September to be able to showcase the change at the call. Sorry about that!

**Comment by matentzn** [comment]
Everyone involved in this issue, I have merged the change one day earlier then I thought, to be able to show the resulting changes at the OFOC call. If you do find any problems as a result of this change, please open an issue and we will deal with it asap.

**Pull request merged**